packages/web-client/app/helpers/format-amount.ts

```diff
@@ -14,9 +14,18 @@ export function formatAmount(
     return '';
   }
 
+  let amountWithoutFloatingPointRoundingError = parseFloat(
```

Comment:
Looks like `spend-to-usd` sometimes produces floats like `23.240000000000002` which will mess up the currency display and show "$23.240000000000000 USD" in this case. However I'm not sure this is the right place to handle this, maybe a better place would be a function in the SDK from where all number/currency related helpers could import.